lib/rubygems/gemcutter_utilities.rb

```diff
@@ -267,6 +269,27 @@ def get_scope_params(scope)
     scope_params
   end
 
+  def get_mfa_params(email, password)
+    mfa_level = get_user_mfa_level(email, password)
+    params = {}
+    if mfa_level == "ui_only" || mfa_level == "ui_and_gem_sign"
+      selected = ask "Would you like to enable MFA for this key? [y/N]"
```

Comment:
To have consistency with the UI (and to increase the use of MFA), maybe we can have the default as Y instead of N.

Reply:
Fantastic idea. Updated in https://github.com/rubygems/rubygems/pull/5305/commits/0b636f690275d9e4a1072e9a7b88ec18bb83d9dd